client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2525,33 +2525,39 @@ def get_fps_for_current_context():
 
 
 def get_frame_range(include_animation_range=False):
-    """Get the current folder frame range and handles.
+    """Get the current task frame range and handles.
 
     Args:
         include_animation_range (bool, optional): Whether to include
             `animationStart` and `animationEnd` keys to define the outer
             range of the timeline. It is excluded by default.
 
     Returns:
-        dict: Folder's expected frame range values.
+        dict: Task's expected frame range values.
 
     """
 
     # Set frame start/end
     project_name = get_current_project_name()
     folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
+
     folder_entity = ayon_api.get_folder_by_path(project_name, folder_path)
-    folder_attributes = folder_entity["attrib"]
+    task_entity = ayon_api.get_task_by_name(
+            project_name, folder_entity["id"], task_name
+        )
```

Comment:
It would be great if we could get to this task with 1 query instead of first requiring to query the current folder by name just for its id. @iLLiCiTiT any possibilities to do that?

Reply:
Not right now, but I agree.

Reply:
should we create an issue to track that ?